src/ssl.c

```diff
@@ -145,6 +140,9 @@
 #endif
 #endif /* !WOLFCRYPT_ONLY || OPENSSL_EXTRA */
 
+#define WOLFSSL_EVP_INCLUDED
+#include "wolfcrypt/src/evp.c"
```

Comment:
With the include remaining here the build process for evp.c is not separated. That's ok, and may have not been the goal of this PR, just placing this comment incase it was part of the goal.